tests/backend/test_states.py

```diff
@@ -377,11 +385,112 @@ def test_number_expectation_four_modes(self, setup_backend, tol, batch_size):
         state = backend.state()
         nbar = np.sinh(r) ** 2
         assert np.allclose(
-            state.number_expectation([0, 1, 2, 3]), (2 * nbar ** 2 + nbar) ** 2, atol=tol, rtol=0
+            state.number_expectation([0, 1, 2, 3]),
+            (2 * nbar ** 2 + nbar) ** 2,
+            atol=tol,
+            rtol=0,
         )
-        assert np.allclose(state.number_expectation([0, 1 ,3]), nbar * (2 * nbar ** 2 + nbar), atol=tol, rtol=0)
-        assert np.allclose(state.number_expectation([3, 1, 2]), nbar * (2 * nbar ** 2 + nbar), atol=tol, rtol=0)
+        assert np.allclose(
+            state.number_expectation([0, 1, 3]),
+            nbar * (2 * nbar ** 2 + nbar),
+            atol=tol,
+            rtol=0,
+        )
+        assert np.allclose(
+            state.number_expectation([3, 1, 2]),
+            nbar * (2 * nbar ** 2 + nbar),
+            atol=tol,
+            rtol=0,
+        )
+
+
+class TestParityExpectation:
+
+    @pytest.mark.backends("fock", "tf")
+    def test_parity_fock(self, setup_backend, tol, batch_size):
```

Comment:
Hi @Lucaman99, thanks so much for this PR! :blush: just leaving a small suggestion: could you add a one-line description as docstring for each test case?

Reply:
Will do!